pandas/core/arrays/string_.py

```diff
@@ -525,7 +525,10 @@ def _reduce(
         self, name: str, *, skipna: bool = True, axis: AxisInt | None = 0, **kwargs
     ):
         if name in ["min", "max"]:
-            return getattr(self, name)(skipna=skipna, axis=axis)
+            result = getattr(self, name)(skipna=skipna, axis=axis)
+            if kwargs.get("keepdims"):
```

Comment:
Could you include `keepdims` in the `_reduce` signature instead?

Reply:
updated